### PR TITLE
Add Riteby Hindew, the Faun Bard

### PR DESCRIPTION
**Pull request by AgentAntelope**

### CODE REVIEW

File: pals/riteby.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+* Escape Artist
+* Skill Versatility
+* Ritual Caster
+  * Glib Limerick
+  * Silence
+
+#### Equipment
+* Dagger (3)
+* Leather Armor
+* Leather Armour
```
Comment (malpinder) [inline pals/riteby.md]: One set of leather armor is probably enough. (Unless the other is something more _amourous_? Ahahaha do you get it)

File: pals/riteby.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+* Lyre
+* Shortbow
+* Silk Rope
```
Comment (malpinder) [inline pals/riteby.md]: Length?

File: pals/riteby.md (hunk 1 above)
Comment (AgentAntelope) [reply pals/riteby.md]: The other one is to protect against Americans!

I'll fix it

Comment (varjmes) [reply pals/riteby.md]: good joke

File: pals/riteby.md [hunk 3]
```diff
@@ -49,9 +65,9 @@ Common, Elven
...
 * Dagger (3)
-* Leather Armour
+* Cloth Armour
 * Lyre
 * Shortbow
 * Silk Rope (50 feet)
-* Standard Kit
-* 6 gp
+* Standard Kit (but without hemp rope)
+* 7 gp
```
Comment (AgentAntelope) [inline pals/riteby.md]: Is this the right amount of gold now?

**Comment by AgentAntelope** [comment]
But my 💰 !

**Comment by malpinder** [comment]
You have 65gp, if you want to spend it just edit the file 😛

**Pull request merged**